express-get-json/server.js

```diff
@@ -0,0 +1,37 @@
+import express from 'express';
+
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95,
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100,
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92,
+  },
+};
+
+const app = express();
+const array = [];
```

Comment:
It's important to put this in the `app.get` arrow function. If it's global then every request will modify it. Try hitting the endpoint twice using httpie, and you will see that the second time you get 6 grades instead of just 3.